Docker/ProxyNginx.conf

```diff
@@ -0,0 +1,41 @@
+upstream dash-handler {
+  server 127.0.0.1:9002;
+}
+
+server {
+  listen 80;
+  server_name dash.example.com;
```

Comment:
Actually, I guess we can make another `nginx` image, like `dash-annotations-nginx`, which will be built on top of this one.